gatsby-site/gatsby-node.js

```diff
@@ -175,3 +175,11 @@ const fileNodeFromRemoteFile = async (helpers, remoteFile) => {
   })
   return fileNode.id
 }
+
+/**
+ * @param {string} input
+ * @returns {string}
+ */
+const parseName = input => {
+  return input.replace(/sunrise(\s+movement)?/i, "").trim()
+}
```

Comment:
Is this because some hub names can already contain Sunrise?

Reply:
It is